src/integrations/misc/open-file.ts

```
@@ -29,12 +29,14 @@ export async function openFile(filePath: string, options: OpenFileOptions = {})
 	try {
 		// Get workspace root
 		const workspaceRoot = vscode.workspace.workspaceFolders?.[0]?.uri.fsPath
-		if (!workspaceRoot) {
-			throw new Error("No workspace root found")
-		}
 
-		// If path starts with ./, resolve it relative to workspace root
-		const fullPath = filePath.startsWith("./") ? path.join(workspaceRoot, filePath.slice(2)) : filePath
+		// If path starts with ./, resolve it relative to workspace root if available
+		// Otherwise, treat it as an absolute path
```

Not sure about treating it as an absolute path. Maybe we should throw an error still if no workspace root, but only for relative paths?

---

I don't understand what the problem with this is.

---

Absolute paths don't start with `./`